database/migrations/2014_03_12_000000_create_users_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
             $table->increments('id');
             $table->string('name');
             $table->string('twitter_id');
+            $table->string('avatar')->nullable()->default(null);
```

Comment:
hm. Was avatar saved just by hashing the twitter id or something before? If so, my one concern with adding it to the DB is that it introduces a place the DB can be out of sync with the filesystem